README.md

```diff
@@ -54,6 +52,70 @@ VitePWA({
 })
 ```
 
+### Prompt for new content 
+
+![](https://user-images.githubusercontent.com/11247099/111190584-330cf880-85f2-11eb-8dad-20ddb84456cf.png)
+
+```ts
+// main.ts
+import { registerSW } from '@virtual/pwa-register'
+
+const updateSW = registerSW({
+  onNeedRefresh() {
+    // show a prompt to user
+  },
+  onOfflineReady() {
+    // ...
+  },
+})
+```
+
+```ts
+// when user clicked the "refresh" button
+updateSW()
+// the page will reload and the up-to-date content will be served.
+```
+
+You can find an example written in Vue 3: [ReloadPrompt.vue](./example/src/ReloadPrompt.vue).
```

Comment:
broken link, should be: `[ReloadPrompt.vue](./examples/vue-basic/src/ReloadPrompt.vue)`

Reply:
I'll fix it